src/ext.rs

```diff
@@ -1083,6 +1084,12 @@ impl UdpSocketExt for UdpSocket {
                mreq)
     }
 
+    #[cfg(target_os = "nto")]
+    fn join_multicast_v6(&self, _: &Ipv6Addr, _: u32)
+                         -> io::Result<()> {
+        Err(io::Error::new(io::ErrorKind::Unsupported, "Not supported by QNX Neutrino"))
```

Comment:
Perhaps make the error strings more closely match what libstd emits: lowercase and generic - "not supported by platform"?

Reply:
Fixed as proposed.

Reply:
Please do the other error at line 1116 too.